src/login.js

```diff
@@ -47,7 +47,7 @@ const login = async function (server='http://localhost:3000', user='liveness', p
     } catch (e) {
       await page.screenshot({ path: `${ commonSetup.SCREENSHOTS_DIR_PATH }/login-failed.png` });
       await browser.close()
-      throw new AutomationError('login-logout-failed', {previous: e})
+      //throw new AutomationError('login-logout-failed', {previous: e})
```

Comment:
Doesn't this ignore more than we expected? If, for example, the avatar doesn't come up, the exception would also be ignored.
Further, the browser will be closed two times.
Is it necessary to create a screenshot when only logout fails?

Reply:
Fixed it, pls review again